arlas-core/src/main/java/io/arlas/server/model/RasterTileURL.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to Gisaïa under one or more contributor
+ * license agreements. See the NOTICE.txt file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Gisaïa licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.arlas.server.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RasterTileURL {
+    @JsonProperty(value = "url", required = true)
+    public String url = null;
+
+    @JsonProperty(value = "id_path", required = true, defaultValue = "id")
+    public String idPath = "id";
+
+    @JsonProperty(value = "min_z", required = false)
+    public int min_z = 0;
+
+    @JsonProperty(value = "max_z", required = false)
+    public int max_z = 18;
+
+    @JsonProperty(value = "check_geometry", required = false)
+    public boolean check_geometry = false;
+
+    public RasterTileURL(){}
+    public RasterTileURL(String url, int min_z, int max_z,boolean check_geometry){
+        this.url = url;
+        this.min_z = min_z;
+        this.max_z = max_z;
```

Comment:
check_geometry attribute is not set in the constructor

Reply:
Indeed. Thank you.